10/umbraco-cms/tutorials/multilanguage-setup.md

```diff
@@ -51,10 +51,28 @@ To change the default language of a website:
 
 1. Go to **Languages** in the **Settings** tree.
 2. Select the language you want to set as the new default language.
-3.  Toggle **Default Language**.\\
+3. Toggle **Default Language**.
 
     <figure><img src="images/change-default-language-v10.png" alt=""><figcaption></figcaption></figure>
+
 4. Click **Save**.
+5. Open the `appsettings.json` file using an IDE.
+6. Update the `Umbraco:CMS:Global:DefaultUILanguage` value to the desired language code. For example:
+
+    ```json
+    {
+    "Umbraco": {
+        "CMS": {
+        "Global": {
+            "DefaultUILanguage": "fr"
+            }
+          }
+        }
+    }
+
+    ```
+
+7. Save the `appsettings.json` file.
```

Comment:
To be honest I don't think this is necessary to have here 🤔

This section is just about adding multiple languages in the settings section, which is used for having different languages displayed on the frontend.

However, the DefaultUILanguage is for changing the language used in the backoffice, so adding that setting should not be relevant in this context 😊

I hope it makes sense!

Reply:
@jonat123 I agree, it's slightly a misfit for this article. 🙈

Would it make sense to add the info in the [Language Variants](https://docs.umbraco.com/umbraco-cms/fundamentals/backoffice/variants) article as a section at the end? 🤔

I think its important to have this info somewhere in the docs, though..

Reply:
I think the problem is that this is not related to Language variants 😊

Also this is already documented here: https://docs.umbraco.com/umbraco-cms/reference/configuration/globalsettings#default-ui-language :-)

Reply:
Ummm okay.. If it's not needed, I'll close this PR then 💪